launchpad_backend/launchpad_server/routes/startup_data.py

```diff
@@ -2,72 +2,749 @@
 
 application = {
     "tableName": "application",
-    "index": "indexId", # Similar to primary key
+    "index": "applicationId", # Similar to primary key
     "records": [{
         "applicationId": 1, 
         "resume": "figure out how to store pdf lol", 
         "coverLetter": "pdf again",
-        "Status": "Reviewed or Applied or Interview Requested",
-        "logo": "img url",
+        "Status": "Reviewed",
+        "logo": "https://upload.wikimedia.org/wikipedia/commons/thumb/4/44/Microsoft_logo.svg/2048px-Microsoft_logo.svg.png",
         "postingId": 1,
         "userId": 1
+    },
+    {
+        "applicationId": 2, 
+        "resume": "figure out how to store pdf lol", 
+        "coverLetter": "pdf again",
+        "Status": "Interview Requested",
+        "logo": "https://seeklogo.com/images/S/scotiabank-logo-D2F1AF87B5-seeklogo.com.png",
+        "postingId": 9,
+        "userId": 1
+    },
+    {
+        "applicationId": 3, 
+        "resume": "figure out how to store pdf lol", 
+        "coverLetter": "pdf again",
+        "Status": "Applied",
+        "logo": "https://upload.wikimedia.org/wikipedia/commons/thumb/4/44/Microsoft_logo.svg/2048px-Microsoft_logo.svg.png",
+        "postingId": 2,
+        "userId": 2
     }]
 }
 
+# add school column
 user = {
     "tableName": "user",
     "index": "userId", 
     "records": [{
         "userId": 1,
-        "email": "michael.cera@yahoo.com", 
-        "password": "superbad", 
-        "firstName": "Mikey", 
-        "lastName": "C", 
-        "year": "1st/2nd/3rd or Graduated", 
+        "email": "ash.ketchum@gmail.com", 
+        "password": "Pikachu@123", 
+        "firstName": "Ash", 
+        "lastName": "Ketchum", 
+        "year": "4th", 
         "program": "Computer Science",
+        "school": "Ryerson University",   # Leave key pair value blank if school not specify
         "address": {        # If address is not specified for a record, do not include this key-value pair in the dictionary
-            "streetAddress": "123 example st",
-            "postalCode": "000 000",
-            "province": "ON"
+            "streetAddress": "",
+            "postalCode": "",
+            "province": ""
         },
-        "phoneNumber": "number",  # If number is not specified for a record, do not include this key-value pair in the dictionary
+        "phoneNumber": "",  # If number is not specified for a record, do not include this key-value pair in the dictionary
```

Comment:
left value blank instead of deleting/removing key/value pair for data consistency